packages/feedback-spa/e2e/tsconfig.json

```
@@ -1,9 +1,9 @@
 {
-  "extends": "../tsconfig.json",
+  "extends": "../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "../out-tsc/e2e",
     "module": "commonjs",
-    "target": "es5",
+    "target": "es2018",
```

```suggestion
    "target": "es2020",
```

https://angular.io/guide/migration-update-module-and-target-compiler-options
@deshmukhmayur @riginoommen if we are upgrading shouldn't we also support modern browsers also with their full capabilities?

---

As long as that doesn't break support for older version, I don't see a problem with doing that.

---

@hybridx  can u plz confirm on this ?

---

For now we can keep it as es2018 because it would take a bit of time to test the changes on other browsers